staging/src/k8s.io/kubectl/pkg/cmd/debug/debug.go

```diff
@@ -120,12 +136,17 @@ func addDebugFlags(cmd *cobra.Command, opt *DebugOptions) {
 	cmd.Flags().BoolVar(&opt.ArgsOnly, "arguments-only", opt.ArgsOnly, i18n.T("If specified, everything after -- will be passed to the new container as Args instead of Command."))
 	cmd.Flags().BoolVar(&opt.Attach, "attach", opt.Attach, i18n.T("If true, wait for the Pod to start running, and then attach to the Pod as if 'kubectl attach ...' were called.  Default false, unless '-i/--stdin' is set, in which case the default is true."))
 	cmd.Flags().StringVarP(&opt.Container, "container", "c", opt.Container, i18n.T("Container name to use for debug container."))
+	cmd.Flags().StringVar(&opt.CopyTo, "copy-to", opt.CopyTo, i18n.T("Create a copy of the target Pod with this name."))
+	cmd.Flags().BoolVar(&opt.CopyLabels, "copy-labels", opt.CopyLabels, i18n.T("When used with '--copy-to', specifies whether labels should also be copied. Note that copying labels may cause the copy to receive traffic from a service or a replicaset to kill other Pods."))
+	cmd.Flags().BoolVar(&opt.DeleteOld, "delete-old", opt.DeleteOld, i18n.T("When used with '--copy-to', delete the original Pod"))
```

Comment:
Maybe `--replace` will better reflect that intention?

Reply:
good call